test/e2e/lib/framework/createTest.ts

```diff
@@ -96,6 +102,11 @@ class TestBuilder {
       declareTest(this.title, setups[0].factory(setupOptions), runner)
     }
   }
+
+  private rumInit: (options: RumSetupOptions) => void = (options) => {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+    ;(window.DD_RUM as any).init(options)
```

Comment:
It seems that `test/e2e/lib/types/global.ts` is not used.  To avoid some type casts, what about importing it somewhere in E2E tests? Else, if we don't want to define those types globally, we could pass a correctly typed DD_RUM argument to `rumInit` (and remove `global.ts`):

```ts
private rumInit: (rumGlobal: RumGlobal, options: RumSetupOptions) => void = (options) => {
   rumGlobal.init(options)
}
```

Reply:
the issue was more on duplicated type between options and configuration, situation should be better now